src/main.rs

```diff
@@ -116,12 +38,12 @@ pub fn main() {
         Err(e) => {
             eprintln!("Error during startup: {}", e);
             error!("Error during startup: {}", e);
-            match e.raw_os_error() {
-                Some(13) => {
-                    startup_helpers::report_permission_error();
-                },
-                _ => eprintln!("Something else!")
-            }
+            if cfg!(target_os = "unix" ) {
+                match e.raw_os_error() {
+                    Some(13) => report_permission_error(),
+                    _        => eprintln!("Something else!")
+                }
+            };
```

Comment:
I added this `if` to prevent `report_permission_error` from being called, unless the `target_os = "unix"`.